### DISCUSSION TITLE
¿Como hago el inicio de Usarios y Voluntarios?

### DISCUSSION THREAD

**Discussion by SaryNotfound** [Curcolor/COMMUNITY#8] [Q&A] [answered]
¿Hacemos algo así en la sesión de inicio como donante o como recolector? O como decía Jorge de un apartado en configuración de usuario

![ejemplo](https://github.com/user-attachments/assets/b5057f9c-189f-46d3-91ac-eac1336b296c)

**Comment by Curcolor** [comment]
A mi en lo personal me gustaría que fuese así como en la imagen, meterse en configuración y cambiar tu estado sería raro, creo yo.

**Comment by JUNIORRDSR** [comment]
> [!IMPORTANT]
>yo creo que desde la configuracion por que asi lo estoy poniendo en el diagrama de casos de uso, aunque no c si se nos facilite hacer que al crear el usuario se ponga de una vez el ser voluntariado pq asi nos facilitaria una tabla para ellos solos en las bases de datos

**Reply by Curcolor** [reply]
Pues para es algo rato tener que meterse a configuración para cambiarte de "estado", estaría bien tenerlo seperado para facilitarnos la vida a la hora de programar.

**Reply by SaryNotfound** [reply, marked as answer]
Yo lo colocaría en el inicio y así el inicio de sesión sería uno para cada uno, ya que son objetivos diferentes.

**Reply by JUNIORRDSR** [reply]
> [!NOTE]
> Bueno modificamos los casos de uso entonces :money_mouth_face:
